### PR TITLE
Update readme with installation and usage instructions #5

### LINKED ISSUE #5: Quick start documentation
Really keen to try out the script and also contribute if needed / possible from my end (skill level wise). 

But I'm missing any and all information how to use the script. 
Setup and build have worked fine, but running the script just executes and does not return any information (and therefore hints regarding what flags etc I need to be giving it).

Would be great to have that in the readme!

### PR DESCRIPTION
**Pull request by tjex**
resolves: #5

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -14,3 +14,44 @@ Configuration
   view ADDRESS = address, address2, city, state, zip, country
   view PHONE = phone, workphone, mobile, other
   view OTHER = nick, url, notes
+
+Install
+------------------
+
+:: 
+
+  git clone git@github.com:jspricke/python-abook.git
```
Comment (jspricke) [inline README.rst]:
```suggestion
  git clone https://github.com/jspricke/python-abook.git
```

**Review by jspricke — COMMENTED** [review]
Maybe adopt the installation from https://github.com/jspricke/python-remind

File: README.rst (hunk 1 above)
Comment (tjex) [reply README.rst]: ups! thanks

**Comment by jspricke** [comment]
Thx!

**Comment by tjex** [comment]
no worries :) thanks for the scripts!

**Pull request merged**